Add template assign notice (#1083)
shopware/devdocs: source/developers-guide/plugin-quick-start/index.md

Now we want to inject our own template directory to Shopware using the `addTemplateDir` method. It takes a path of a template directory as parameter. Whenever rendering a template, Shopware will now automatically check your `View` directory for extensions and load them dynamically.

<div class="alert alert-warning">
<strong>Be careful with sensitive data!</strong>

> Please be aware not to assign sensitive data to the Template on events like global events like PostDispatch. The view variables are shared accross multiple renders. The JsonRenderer as example will sent all variables to the client.

Reviewer: ```suggestion
Please make sure that you not assign sensitive data to the template on global events like PostDispatch. The view variables are shared accross multiple renders. The JsonRenderer for example will send all variables to the client.
```
would should also link to the event page on "PostDispatch"
